app/config/themes/default.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
1. Yes, mostly for security (but also for performance and correctness) I opted to not process URLs via a query parameter.
2. There are 4 kinds of 'custom installs':
   a. Own source code & server - can just modify the JSON at will 
   b. Own server but prefer working with prebuilt docker images - ADD/COPY file is simplest (see for example https://hub.docker.com/_/nginx/)
   c. 'Heroku' style deploy from git repo - in a way it's similar to (a). 
        Either way it's not much of a hassle if you only need theming, as you can fork & add a theme file using GitHub's UI alone.
   d. Partner that doesn't own code or servers - for this case you can specify a predefined theme as a query parameter.